subsys/mgmt/mcumgr/Kconfig

```diff
@@ -131,6 +131,56 @@ config FS_MGMT_DL_CHUNK_SIZE
 
 endif
 
+config FS_MGMT_FILE_STATUS_ENABLED
```

Comment:
`_ENABLED` non needed. For example look at this `IS_ENABLED(CONFIG_FS_MGMT_FILE_STATUS_ENABLED)` would look strange.

Reply:
Done